src/wallet/api/wallet.cpp

```diff
@@ -1161,6 +1225,16 @@ AddressBook *WalletImpl::addressBook() const
     return m_addressBook;
 }
 
+Subaddress *WalletImpl::subaddress() const
```

Comment:
Since the function is const, it should return a const pointer to data it has. It should be different from other functions if other functions are wrong.